tests/functional/behave_features/common/utils/setttings.py

```diff
@@ -8,7 +8,11 @@
 PROD_REPO = "openshift-helm-charts/charts"
 # The prod branch where we store all chart files
 PROD_BRANCH = "main"
-# This is used to find chart certification workflow run id
+# (Deprecated) This is used to find chart certification workflow run id
```

Comment:
Note for reviewers:

This is here to avoid any chicken/egg problems caused by its removal. In my mind, I want to roll this out without removing CERTIFICATION_CI_NAME, find all references to it, replace them, then roll out the removal at a later time.

Maybe an unnecessary caution, but that's the purpose of the deprecation message.